apps/api/src/app/widgets/usecases/initialize-session/initialize-session.usecase.ts

```diff
@@ -28,6 +29,10 @@ export class InitializeSession {
       phone: command.phone,
     });
 
+    if (!application) {
+      throw new ApiException('Please provide a valid app identifier');
+    }
```

Comment:
@AlexVCS I think I found the issue, this if statement should be before the `comandos` variable creation, because the error "Cannot read property _id of undefined" says that in line 23 where it says `application._id` is tried to access, but application does not exist.

What we want to do is to make the if statement **before** this called is called and not after to prevent this exception from happening.

So I believe just moving this if statement to the correct place should fix this :)